tests/connection_e2e_fcr2gcp_test.go

```diff
@@ -0,0 +1,21 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCloudRouter2GcpCreateConnection(t *testing.T) {
+	// retryable errors in terraform testing.
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "../examples/fabric-cloud-router/fcr2gcp",
+	})
+
+	defer terraform.Destroy(t, terraformOptions)
+
+	terraform.InitAndApply(t, terraformOptions)
+	output := terraform.Output(t, terraformOptions, "fcr2gcp_connection_result")
```

Comment:
Output is named "connection_result" in main.tf of example script. Needs to match in order to pass the test after run.

Reply:
Ack. I have made all the files consistent and updated to a generic "connection_result" in all places.